oak-commons/src/test/java/org/apache/jackrabbit/oak/commons/collections/CollectionUtilsTest.java

```diff
@@ -308,6 +308,68 @@ public void testIntersectionWithNoCommonElements() {
         Assert.assertEquals(Collections.EMPTY_SET, CollectionUtils.intersection(set1, set2));
     }
 
+    @Test
+    public void testSymmetricDifferenceWithNonEmptySets() {
+        final Set<String> set1 = Set.of("a", "b", "c");
+        final Set<String> set2 = Set.of("b", "c", "d");
+
+        final Set<String> result = CollectionUtils.symmetricDifference(set1, set2);
+
+        final Set<String> expected = Set.of("a", "d");
+        Assert.assertEquals(expected, result);
+    }
+
+    @Test
+    public void testSymmetricDifferenceWithEmptySet() {
+        final Set<String> set1 = Set.of("a", "b", "c");
+        final Set<String> set2 = new HashSet<>();
+
+        final Set<String> result = CollectionUtils.symmetricDifference(set1, set2);
+
+        final Set<String> expected = Set.of("a", "b", "c");
+        Assert.assertEquals(expected, result);
+    }
+
+    @Test
+    public void testSymmetricDifferenceWithBothEmptySets() {
+        final Set<String> set1 = new HashSet<>();
+        final Set<String> set2 = new HashSet<>();
+
```

Comment:
Same here.

Reply:
done in https://github.com/apache/jackrabbit-oak/pull/1924/commits/78590cb471e03f626fd91f904dd507171759090b